groups/groups.yaml

```diff
@@ -336,63 +165,6 @@ groups:
   #     WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
   #
 
-  - email-id: k8s-infra-rbac-cert-manager@kubernetes.io
-    name: k8s-infra-rbac-cert-manager
-    description: |-
-      Grants access to the `namespace-user` role in the `cert-manager` namespace on the `aaa` cluster
-    settings:
-      ReconcileMembers: "true"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
-    members:
-      - ameukam@gmail.com
-      - cblecker@gmail.com
-      - james@munnelly.eu
-      - thockin@google.com
-
-  - email-id: k8s-infra-rbac-k8s-io-canary@kubernetes.io
-    name: k8s-infra-rbac-k8s-io-canary
-    description: |-
-      Grants access to the `namespace-user` role in the `k8s-io-canary` namespace on the `aaa` cluster
-    settings:
-      ReconcileMembers: "true"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
-    members:
-      - k8s-infra-rbac-k8s-io-prod@kubernetes.io
-
-  - email-id: k8s-infra-rbac-k8s-io-prod@kubernetes.io
-    name: k8s-infra-rbac-k8s-io-prod
-    description: |-
-      Grants access to the `namespace-user` role in the `k8s-io-prod` namespace on the `aaa` cluster
-    settings:
-      ReconcileMembers: "true"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
-    members:
-      - antonio.ojea.garcia@gmail.com
-      - bentheelder@google.com
-      - cblecker@gmail.com
-      - davanum@gmail.com
-      - nikhitaraghunath@gmail.com
-      - spiffxp@gmail.com
-      - thockin@google.com
-
-  - email-id: k8s-infra-rbac-perfdash@kubernetes.io
-    name: k8s-infra-rbac-perfdash
-    description: |-
-      Grants access to the `namespace-user` role in the `perfdash` namespace on the `aaa` cluster
-    settings:
-      ReconcileMembers: "true"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
-    members:
-      - jkaniuk@google.com
-      - jprzychodzen@google.com
-      - jupblb@google.com
-      - maciejborsz@google.com
-      - marcelzieba@google.com
-      - mmatejczyk@google.com
-      - shyam123.jvs95@gmail.com
-      - wojtekt@google.com
-      - zawodny@google.com
-
   - email-id: k8s-infra-rbac-triageparty-cli@kubernetes.io
```

Comment:
Could be move to `groups/sig-cli` ? (the same is done for sig-release)

Reply:
wasn't sure if this would conflict with the inflight triageparty-cli PR or not so left it for next round of PRs